Src/library/base/elks/kernel/raw_file.e

```diff
@@ -20,7 +20,7 @@ create
 	make, make_with_name, make_with_path,
 	make_open_read, make_open_write, make_open_append,
 	make_open_read_write, make_create_read_write,
-	make_open_read_append
+	make_open_read_append, make_temporary_open, make_temporary_open_with_prefix
```

Comment:
I would group the temporary* creation proc on the next line.

Reply:
ok good.